.github/workflows/reusable-build-test-release.yml

```diff
@@ -657,7 +658,7 @@ jobs:
         id: run-tests
         env:
           ARGO_TOKEN: ${{ steps.get-argo-token.outputs.argo-token }}
-        uses: splunk/wfe-test-runner-action@v1.6
+        uses: splunk/wfe-test-runner-action@v1.6.1
```

Comment:
maybe let's leave 1.6? It will use 1.6.1 automatically and leaves a place for fixes. Refers to all changes for
`uses: splunk/wfe-test-runner-action@v1.6 `

Reply:
We are manually releasing the version of wfe-test-runner-action, so releasing v.1.6.1 of action will not update the code for v1.6

Reply:
I mean syntax
`uses: splunk/wfe-test-runner-action@v1.6`
will use the newest version of 1.6.x `wfe-test-runner-action` so no need to explicitly set it to 1.6.1

Reply:
After discussion, it should stay as 1.6.1